ts/client/src/accounts/mangoAccount.ts

```diff
@@ -1432,6 +1438,27 @@ export class PerpPosition {
     );
   }
 
+  public getLiquidationPrice(group: Group, mangoAccount: MangoAccount): I80F48 {
```

Comment:
maybe return I80F48 | null or something, so there doesn't need to be a I80F48 marker value for "none"?